pkg/manager/setconfig.go

```diff
@@ -19,7 +19,9 @@ import (
 	"github.com/onosproject/onos-config/pkg/events"
 	"github.com/onosproject/onos-config/pkg/store"
 	"github.com/onosproject/onos-config/pkg/store/change"
-	types "github.com/onosproject/onos-config/pkg/types/change/device"
+	devicechangetypes "github.com/onosproject/onos-config/pkg/types/change/device"
+	networktypes "github.com/onosproject/onos-config/pkg/types/change/network"
+	devicepb "github.com/onosproject/onos-topo/pkg/northbound/device"
```

Comment:
I think this should be `devicetopo` to give its purpose

Reply:
same

Reply:
then i think you should be consistent - `networktypes` should be `networkchangetypes`

Reply:
sure